packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -793,8 +807,12 @@ export class ColorPicker
                 aria-valuenow={color?.round().hue() || DEFAULT_COLOR.round().hue()}
                 class={{ [CSS.scope]: true, [CSS.hueScope]: true }}
                 onKeyDown={this.handleHueScopeKeyDown}
+                onPointerDown={this.handleHueSliderPointerDown}
                 role="slider"
-                style={{ top: `${hueTop}px`, left: `${hueLeft}px` }}
+                style={{
+                  top: `${hueTop - DIMENSIONS.scopeNode / 2}px`,
```

Comment:
since a few places are dividing a value by the scopeNode and converting to px maybe convert that into a function for reuse.